components/atom/textarea/src/index.js

```diff
@@ -8,8 +8,23 @@ const SIZES = {
   LONG: 'long'
 }
 
-const AtomTextarea = ({onChange, size, value, ...props}) => {
-  const className = cx(BASE_CLASS, `${BASE_CLASS}--${size}`)
+const ERROR_STATES = {
+  ERROR: 'error',
+  SUCCESS: 'success'
+}
+
+const getErrorStateClass = errorState => {
+  if (errorState) return `${BASE_CLASS}--${ERROR_STATES.ERROR}`
+  if (errorState === false) return `${BASE_CLASS}--${ERROR_STATES.SUCCESS}`
```

Comment:
False or Falsish ?!

Because `undefined !== false`

Reply:
if errorState is true --> there is error. if errorState is false -->  there is not error and it is success. Otherwise there is not error and there is not success.

Reply:
it is not the best logic (in my opinion) but there are other components that share this logic.